### PR TITLE
Fix compartment report mapping

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: brayns/parameters/GeometryParameters.h [hunk 1]
```diff
@@ -261,7 +261,7 @@ class GeometryParameters : public AbstractParameters
     std::string _circuitReport;
     double _circuitStartSimulationTime;
     double _circuitEndSimulationTime;
-    double _circuitSimulationStep;
+    double _circuitSimulationStep{0};
```
Comment (favreau) [inline brayns/parameters/GeometryParameters.h]: Why is this required?

Comment (tribal-tec) [reply brayns/parameters/GeometryParameters.h]: because it's uninitialized?
